### DISCUSSION TITLE
Install from manifest file

### DISCUSSION THREAD

**Discussion by dave992** [microsoft/vcpkg#27157] [Q&A] [answered]
Hi all,

I would like to know if it is possible to install the dependencies listed in a manifest without running CMake. 
For example: `vcpkg install << vcpkg.json`

Thanks in advance!

**Comment by autoantwort** [comment, marked as answer]
Run `vcpkg install` in the folder where the `vcpkg.json` file is located.

**Reply by dave992** [reply]
Sometimes it's incredible how easy it is to overlook the simplest solutions, I've been stuck in trying to force it in manifest mode specifying the project folder containing the manifest and/or providing the manifest file as input to the install command.

It worked, and all it took was a single `cd` command, thanks!

**Reply by dg0yt** [reply]
Note that a plain `vcpkg install` is unaware of your cmake build directory. But it is good enough to populate the binary cache for later reuse.

**Comment by Neumann-A** [comment]
An alternative to the above (or below) is to pass `--x-manifest-root=<dir_containing_manifest>` or just compare against what is running in cmake: https://github.com/microsoft/vcpkg/blob/083f103cfb6ff4a05d6c49b6a2231965e08b6e2b/scripts/buildsystems/vcpkg.cmake#L507-L515
